.cascade-code/Chapman.edu/_config/global_nav_law.vtl

```diff
@@ -0,0 +1,214 @@
+#set ( $GLOBAL_NAV_LAW_CONFIG = [
```

Comment:
Should this have `.vm` extension? I'm comparing to [this line](https://github.com/chapmanu/cascade-assets/pull/281/files#diff-9eb228b5664fc3bdcb787f84d31f02bbR17).

Reply:
yes. fixed.